docs/ops-manual.md

```diff
@@ -2,55 +2,19 @@
 
 ## SSHing to a live docker container
 
-To SSH into a container, install the `cloudfoundry-cli`. Example using `asdf`:
-
-```bash
-$ asdf plugin add cf
-$ asdf cf install latest
-$ asdf global cf latest
-$ cf --version
-cf version 8.3.0+e6f8a85.2022-03-11
-```
-
-You'll need a PaaS account that has access to the
-`tra-dev/tra-test/tra-production` space. Sign into your account using SSO:
-
-```bash
-$ cf login -a api.london.cloud.service.gov.uk --sso
-API endpoint: api.london.cloud.service.gov.uk
-
-Temporary Authentication Code ( Get one at https://login.london.cloud.service.gov.uk/passcode ):
-Authenticating...
-OK
+To SSH into a container, install `kubectl`
 
+- Configure the credentials using the `get-cluster-credentials`, within the Teacher-services-cloud repo, run make command. Example:
```

Comment:
I think we should add 'get-cluster-credentials' to this services Makefile, rather than having to install another repo